Classes/Service/CacheService.php

```diff
@@ -71,6 +80,16 @@ public function flush($force = false)
     }
 
     /**
+     * Flushes caches using the data handler. This should not be necessary any more in the future.
+     * @deprecated
+     * @internal
+     */
+    public function flushCachesWithDataHandler()
+    {
+        $this->dataHandler->clear_cacheCmd('all');
```

Comment:
my intention is not to clear further caches, but to call hook subscribers.

Do you have examples of hook subscribers, that only do something with "system" command? If these exist, we may also need to call "pages".

Oh, and "system" has ben deprecated in TYPO3 8.

So, what is your exact use case? which extension(s) subscribe to this hook, that you want to be handled?